portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,108 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+    display: block;
```

Comment:
[2.2.1](https://google.github.io/styleguide/htmlcssguide.html#Indentation): Indent by 2 spaces at a time.

For some reason the Cloud Shell Editor does not currently support formatting CSS files though so the easiest way to do this is by:
1. Selecting the entire document **except** for the correctly indented CSS rules above this one.
1. Outdenting with `Shift` + `Tab` or `Ctrl`/`Cmd` + `[`.

That'll fix most of the rules here from 4 to 2 spaces. You still have some other rules below to fix though that currently have 0 indentation and there are some inner blocks that need to be reduced from 8 to 4 spaces.